frontend/TvCv19.Frontend/ClientApp/projects/shared-caregiver/src/lib/route-helper.ts

```diff
@@ -9,44 +9,75 @@ import { ChatRouteResolverService } from './chat/chat-route-resolver.service';
 import { PatientListComponent } from './patient-list/patient-list.component';
 import { ChangeShiftRouteResolverService } from './change-shift/change-shift-route-resolver.service';
 import { PatientDetailRouteResolverService } from './patient-detail/patient-detail-route-resolver.service';
+import { NursesDoctorsMenuComponent } from './nurses-doctors-menu/nurses-doctors-menu.component';
 
-export function getCaregiverRoute(rootComponent: Type<any>, caregiverRootComponent: Type<any>): Routes {
-    return [
-        { path: '', component: rootComponent }, 
-        { path: ':id', component: caregiverRootComponent, children: [
-            { path: '',
-                component: PhysicianHierarchyComponent,
-                resolve: {
-                    model: CarerRouteResolverService
-                }
+export function getCaregiverRoute(
+  rootComponent: Type<any>,
+  caregiverRootComponent: Type<any>
+): Routes {
+  return [
+    { path: '', component: rootComponent },
+    {
+      path: ':id',
+      component: caregiverRootComponent,
+      children: [
+        {
+          path: '',
+          component: PhysicianHierarchyComponent,
+          resolve: {
+            model: CarerRouteResolverService,
+          },
+        },
+        {
+          path: 'patients',
+          component: PatientListComponent,
+          resolve: {
+            model: CarerRouteResolverService,
+          },
+        },
+        {
+          path: 'menu',
+          component: NursesDoctorsMenuComponent,
```

Comment:
can you rename the component and associated files to CarerMenuComponent
just because its going to used by bedside care givers as well as doctors and nurses

Reply:
renamed everything to CarerMenuComponent and files to carer-menu